colossus/src/main/scala/colossus/protocols/websocket/Websocket.scala

```diff
@@ -166,7 +166,8 @@ object FrameParser {
   }
 }
 
-abstract class WebsocketHandler(context: Context) extends Controller(new WebsocketCodec, ControllerConfig(50, scala.concurrent.duration.Duration.Inf), context) {
+abstract class WebsocketHandler(context: Context)
```

Comment:
Shouldn't this be taking in a configuration, and then passing that to the parent?